docs/auth/microsoft/azure-easyauth.md

```diff
@@ -69,17 +79,25 @@ import { ProxiedSignInPage } from '@backstage/core-components';
 const app = createApp({
   /* highlight-add-start */
   components: {
-    SignInPage: props => (
-      <ProxiedSignInPage {...props} provider="azure-easyauth" />
-    ),
+    SignInPage: props => {
+      if (process.env.NODE_ENV === 'development') {
```

Comment:
I am not sure what difference it makes. but I used 

```      const configApi = useApi(configApiRef);
      if (configApi.getString('auth.environment') === 'development') ```

Reply:
Both methods are fine, depends on what setup you choose. It's covered at the end of https://backstage.io/docs/auth/#sign-in-with-proxy-providers

Perhaps it's best to stick to a single recommendation to avoid confusion, what do you think about switching it over to the config approach @afscrome?